Source/FakeItEasy.Specs/DomainEvents.cs

```diff
@@ -1,8 +1,26 @@
 ﻿namespace FakeItEasy.Specs
 {
+    using System;

```

Comment:
Until now this (test data) file (`DomainEvents.cs`) has a far more specific purpose (in the configuration/factory specs). Do we really want to reuse it in tests regarding the virtual calls in a c'tor?

Also: Shouldn't we give it a name where the intended purpose becomes clearer, anyways?